01week/datatypes.js

```diff
@@ -0,0 +1,70 @@
+1. Write a JavaScript program to display the current day and time.
```

Comment:
put this in a comment `//1. Write a JavaScript program to display the current day and time. `. Without, you will throw an error.